firebase_admin/_utils.py

```diff
@@ -93,8 +93,10 @@ def _get_initialized_app(app):
 
 
 
-def get_app_service(app, name, initializer):
+def get_app_service(app, options, name, initializer):
```

Comment:
I see this is a private module, but adding a new positional argument into the middle of the list seems dangerous. Can this be an optional kwarg?

Reply:
Done